src/store/view-configs/volume-coloring.ts

```diff
@@ -52,8 +52,7 @@ export const defaultVolumeColorConfig = (): VolumeColorConfig => ({
   },
   cvr: {
     enabled: false,
-    flipLightPosition: false,
-    fixedLightPosition: false,
+    lightFollowsCamera: false,
```

Comment:
I would suggest having the light following the camera by default, otherwise the default light location won't make sense.